ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/SketchOnMap/SketchOnMap.qml

```diff
@@ -53,150 +84,70 @@ Item {
                 columns: 2
 
                 Text {
-                    id: geometryText
-                    color: "#ffffff"
-                    text: qsTr("Geometry")
-                    font.pixelSize: 16
+                    id: geometryHeader
                     Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
-                    font.bold: true
                     Layout.columnSpan: 2
+                    text: "Geometry"
+                    color: "white"
+                    font.pixelSize: 16
+                    font.bold: true
                 }
 
-                RoundButton {
+                SketchEditorButton {
                     id: ptButton
-                    radius: 5
-                    Layout.fillWidth: true
-                    Layout.alignment: Qt.AlignHCenter
-
-                    Image {
-                        id: ptImage
-                        source: "iconAssets/point-32.png"
-                        anchors {
-                            horizontalCenter: parent.horizontalCenter
-                            verticalCenter: parent.verticalCenter
-                            verticalCenterOffset: -ptText.height/2
-                        }
-                        width: 20
-                        fillMode: Image.PreserveAspectFit
-                    }
+                    buttonName: "Point"
+                    iconPath: "iconAssets/point-32.png"
```

Comment:
this might need a qrc path instead - do the icons deploy correctly on mobile as-is?

Reply:
It runs as expected without the `qrc` prefix, but I added it in to be safe.